.changelog/4343.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Mobilized base_playbook tasks into objects to ease on the use in various flows.
```

Comment:
mobilized is definitely not the right word

Reply:
Meant modelized